libraries/aws/shadow/cbmc/proofs/Shadow_MatchTopic/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=Shadow_MatchTopic_harness
+
+# The topic length is bounded for thing name because its maximum length is 64KB which
```

Comment:
It is less than 5 minutes. We get full coverage because this bound only affects a strncmp.

Reply:
Because the real maximum length which is 64KB is too long to test, so we bound it to 256 to test. In this case, our coverage is full.

Reply:
It's an estimation, the real maximum loop execution number will be: 256 - 12 ( fixed aws prefix prefix length )= 244, according to our experience before, it would be less than 50, thus we thought it's enough to cover the use case.